ipmiMgrApp/Db/mch_nat.substitutions

```diff
@@ -27,23 +27,23 @@
 file "mch_base.db"
 {
     pattern { id           , fruid }
-            { $(id)Base    , 3     }
+            { Base    , 3     }
```

Comment:
Please explain why this is wrong in the commit message.

Reply:
I feel like the message could be a little bit more direct to the point. Not a specific suggestion, though.

Reply:
Have you taken another look at this?

At least a typo should be resolved: `s/them/then/`.